Makefile

```diff
@@ -1,21 +1,36 @@
-KICKSTART_FILE=centos-7-minimal.ks
-KICKSTART_TEMPLATE=centos-7-minimal.template
+CENTOS_KICKSTART_FILE=centos-7-minimal.ks
+CENTOS_KICKSTART_TEMPLATE=centos-7-minimal.template
+
+RHEL_KICKSTART_FILE=rhel-7-minimal.ks
+RHEL_KICKSTART_TEMPLATE=rhel-7-minimal.template
+
 
 BUILD_DIR=$(shell pwd)/build
-ISO_NAME=live-centos
+CENTOS_ISO_NAME=live-centos
+RHEL_ISO_NAME=live-rhel
 
 HANDLE_USER_DATA=$(shell base64 -w 0 scripts/handle-user-data)
 CERT_GEN=$(shell base64 -w 0 scripts/cert-gen.sh)
 
-default: iso
+default: centos_iso
 
 kickstart:
 	mkdir -p $(BUILD_DIR)
-	touch $(BUILD_DIR)/$(KICKSTART_FILE)
-	handle_user_data='$(HANDLE_USER_DATA)' cert_gen='$(CERT_GEN)' envsubst <  $(KICKSTART_TEMPLATE) > $(BUILD_DIR)/$(KICKSTART_FILE)
 
-iso: kickstart
-	cd $(BUILD_DIR); sudo livecd-creator --config $(BUILD_DIR)/$(KICKSTART_FILE) --logfile=$(BUILD_DIR)/livecd-creator.log --fslabel $(ISO_NAME)
+centos_kickstart: kickstart
+	touch $(BUILD_DIR)/$(CENTOS_KICKSTART_FILE)
+	handle_user_data='$(HANDLE_USER_DATA)' cert_gen='$(CERT_GEN)' envsubst <  $(CENTOS_KICKSTART_TEMPLATE) > $(BUILD_DIR)/$(CENTOS_KICKSTART_FILE)
+
+rhel_kickstart: kickstart
```

Comment:
there seems to me a lot of duplication here. Why do you cut&paste the various commands. I would expect there to be one _iso_ target which does the actual iso creation. Then we have a _centos_iso_ and _rhel_iso_ target which just set the right values for _KICKSTART_FILE_, _KICKSTART_TEMPLATE_ and _ISO_NAME_. Much less duplication and more re-use.